src/site/layouts/vet_center.drupal.liquid

```diff
@@ -74,29 +74,53 @@
 
           </div>
 
-          <h2 id="prepare-for-your-visit" class="vads-u-margin-top--0 vads-u-font-size--lg
+            {% if fieldPrepareForVisit.length > 0 %}
+                <h2 id="prepare-for-your-visit" class="vads-u-margin-top--0 vads-u-font-size--lg
           small-screen:vads-u-font-size--xl vads-u-margin-bottom--2">Prepare for your visit</h2>
-            <div class="vads-u-margin-bottom--3">
-                {% for entity in fieldPrepareForVisit %}
-                    <va-accordion bordered >
-                        <va-accordion-item header="{{ entity.entity.fieldHeader }}" level="3">
-                            {{ entity.entity.fieldRichWysiwyg.processed | drupalToVaPath | outputLinks }}
-                        </va-accordion-item>
-                    </va-accordion>
-                {% endfor %}
-            </div>
+                <p> Click on a topic for more details. </p>
+                <div class="vads-u-margin-bottom--3">
+                    {% for entity in fieldPrepareForVisit %}
+                        <va-accordion bordered id="prepare-for-your-visit-accordion-{{ entity.entity.fieldHeader }}" >
+                            <va-accordion-item id="prepare-for-your-visit-accordion-item-{{ entity.entity.fieldHeader }}" header="{{ entity.entity.fieldHeader }}" level="3">
+                                {{ entity.entity.fieldRichWysiwyg.processed | drupalToVaPath | outputLinks }}
+                            </va-accordion-item>
+                        </va-accordion>
+                    {% endfor %}
+                </div>
+            {% endif %}
 
           <h2 id="in-the-spot-light" class="vads-u-margin-top--0 vads-u-font-size--lg
           small-screen:vads-u-font-size--xl vads-u-margin-bottom--2">In the spotlight at {{ entityLabel }} </h2>
 
-          <h2 id="counseling-services" class="vads-u-margin-top--0 vads-u-font-size--lg
+            {% assign counselingServices = fieldHealthServices | filterByTypeOfCare: "counselling" %}
+            {% if counselingServices.length > 0 %}
+                <h2 id="counseling-services" class="vads-u-margin-top--0 vads-u-font-size--lg
           small-screen:vads-u-font-size--xl vads-u-margin-bottom--2">Counseling services</h2>
+                <p> Click on a service for more details. </p>
+                {% include "src/site/includes/vet_center_services.liquid" with
+                    services = counselingServices
+                %}
+            {% endif %}
```

Comment:
Each of these blocks is the samee, except for the heading label and the type of care filter.

Can we refactor into a separate template?